src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -79,6 +87,14 @@ private static bool TryReadMemoryValueFromFile(string path, out ulong result)
                         result = checked(ulongValue * multiplier);
                         return true;
                     }
+
+                    if (MaxBytes == bytes.AsSpan())
```

Comment:
This unfortunately is not going to do what you want it to do, as == on span behaves differently than == on string.  This one could use Equals with StringComparison, but it'd be better just as:
```C#
if (MaxBytes.SequenceEquals(bytes))
```